tgui/packages/tgui/interfaces/Spellbook.tsx

```diff
@@ -328,7 +320,8 @@ const Randomize = (props) => {
     <Stack fill vertical>
       {points < 10 && <PointLocked />}
       <Stack.Item>
-        Semi-Randomize will ensure you at least get some mobility and lethality and additional 20% spell value.
+        Choosing to randomize will give some bonus points and buy a random selection of spells. This is not reversible, it
+        disables refunding, and will not buy staves, wands, or similar.
```

Comment:
You get cut off there?

```suggestion
        disables refunding, and will not buy staves, wands, or similar magical items.
```